scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua

```diff
@@ -20,76 +22,283 @@ local minionGroup =
     [7] = 25, -- Qnhpemde
 }
 
-entity.onMobEngage = function(mob, target)
+local spellLists =
+{
+    [1] = 5025, -- Fire List
+    [2] = 5026, -- Ice List
+    [3] = 5027, -- Wind List
+    [4] = 5028, -- Earth List
+    [5] = 5029, -- Ltng List
+    [6] = 5030, -- Water List
+    [7] = 5024, -- Light List
+    [8] = 5031, -- Dark List
```

Comment:
This indexes are xi.element.NAME

We will need to make sure this IDs are defined in the DB aswell

Reply:
updated to use real spell lists